src/Flicking.ts

```diff
@@ -495,6 +501,7 @@ class Flicking extends Component<{
     }
 
     this.viewport.destroy(option);
+    this.contentsReadyChecker?.off();
```

Comment:
It is better to use destroy rather than off.

Reply:
I've fixed it, can you check what I've written above?